fs/fs.go

```diff
@@ -346,7 +346,8 @@ func (self *RealFsInfo) GetFsInfoForPath(mountSet map[string]struct{}) ([]Fs, er
 	return filesystems, nil
 }
 
-var partitionRegex = regexp.MustCompile(`^(?:(?:s|xv)d[a-z]+\d*|dm-\d+)$`)
+//update by fzu-huang.Update the filter to match disk such as 'vda'.
```

Comment:
Please remove "update by fzu-huang" everywhere. It is recorded in the git history.